the-collab-lab/tcl-19-smart-shopping-list
# Cu jm add new item

## Description

This PR adds two components: one for adding new items to the database associated with the user's token and another for displaying the list of items as an unordered list.

## Related Issue

Closes #4 

## Acceptance Criteria


## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

![Screenshot (49)](https://user-images.githubusercontent.com/57858821/104708523-aec1f600-56d2-11eb-9e99-9c3ab6ef7228.png)
![Screenshot (48)](https://user-images.githubusercontent.com/57858821/104708526-aff32300-56d2-11eb-8e39-8f98eea9ef96.png)



## Testing Steps / QA Criteria

- From your terminal, pull down this branch with git pull origin cu-jm-add-new-item and check that branch out with git checkout cu-jm-add-new-item
- npm install to install app dependencies
- Fill out the form with a new item, choose an option among the radio buttons and submit
- Navigate to the lists view to view the added list
